app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}. {self.nickname} has bow of the "

```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line